content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -867,6 +873,50 @@ is disabled, `.spec.completions` is immutable.
 Use cases for elastic Indexed Jobs include batch workloads which require 
 scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
 
+### Pod Replacement Policy
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only set `podReplacementPolicy` on Jobs if you enable the `JobPodReplacementPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+{{< /note >}}
+
+By default, the Job controller recreates pods as soon they are either failed or terminating (have a deletion timestamp).
+This means that, at a given time, the number of running Pods for the Jobs can be greater than `parallelism` or, if using Indexed Jobs, more than one running Pod per index, if some of the Pods are terminating.
+
+You may choose to create replacement pods only when the terminating pod is fully terminal (has `status.phase: Failed`). To do this, set the `.spec.podReplacementPolicy: Failed`.
+This will only recreate pods once they are terminated.  
+The default policy is `FailedOrTerminating`, meaning that the control plane creates replacement Pods upon deletion (`DeletionTimestamp != nil`).
```

Comment:
```suggestion
The default policy is `FailedOrTerminating`, meaning that the control plane creates replacement Pods
immediately upon deletion (as soon as the control plane sees that a Pod for this Job has
`deletionTimestamp` set).
```

Reply:
Accepted and removed paragraph about `podFailurePolicy`